README.md

```diff
@@ -145,6 +146,21 @@ To clone it into a folder with a different name than the default repository name
 $ git clone [url] name-of-new-folder
 ```
 
+### I set the wrong remote repository.
```

Comment:
No final period

Reply:
This conforms to every other header. Periods are not normally used in titles anyways.